crates/ruff_python_formatter/src/range.rs

```diff
@@ -0,0 +1,793 @@
+use tracing::Level;
+
+use ruff_formatter::printer::SourceMapGeneration;
+use ruff_formatter::{
+    format, FormatContext, FormatError, FormatOptions, IndentStyle, PrintedRange, SourceCode,
+};
+use ruff_python_ast::visitor::preorder::{walk_body, PreorderVisitor, TraversalSignal};
+use ruff_python_ast::{AnyNode, AnyNodeRef, Stmt, StmtMatch, StmtTry};
+use ruff_python_index::tokens_and_ranges;
+use ruff_python_parser::{parse_tokens, AsMode, ParseError, ParseErrorType};
+use ruff_python_trivia::{
+    has_leading_content, indentation_at_offset, is_python_whitespace, BackwardsTokenizer,
+    SimpleToken, SimpleTokenKind,
+};
+use ruff_source_file::Locator;
+use ruff_text_size::{Ranged, TextLen, TextRange, TextSize};
+
+use crate::comments::Comments;
+use crate::context::{IndentLevel, NodeLevel};
+use crate::prelude::*;
+use crate::statement::suite::DocstringStmt;
+use crate::verbatim::{ends_suppression, starts_suppression};
+use crate::{format_module_source, FormatModuleError, PyFormatOptions};
```

Comment:
Out of curiosity, why not `use crate::{comments::Comments, prelude::*, format_module_source, ...};` instead? (And similar above.)

Reply:
I rarely write the use statements by hand. This is what RustRover generates (after running Organize imports. I would need to look into the settings to see if I can fine tune the behavior 

I personally would prefer one use per line (and sorted) because I find merging use conflicts rather annoying and my IDE collapses the use statements anyway. 

To sum it up: it's not intentional. I don't write use statements by hand